### ISSUE TITLE
Method with name `empty` is always reported as error

### ISSUE THREAD

**Issue by dlakomski** [squizlabs/PHP_CodeSniffer#3609]
**Describe the bug**
PHPCS considers methods with name `empty` to be incorrect

**Code sample**
```php
<?php

declare(strict_types=1);

namespace App;

class Foo
{
    public function empty()
    {
        return;
    }
}
```

**Custom ruleset**
No custom rules. Just PSR12 standard

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs --standard=PSR12 test.php`
3. See error message displayed
```
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 9 | ERROR | Expected "function abc(...)"; found "function abc(...)"
----------------------------------------------------------------------
```

If you change method name to basically anything else - `empty1`, `createEmpty` etc - there is no error

**Expected behavior**
No complains.

**Versions (please complete the following information):**
 - OS: Ubuntu 20.04
 - PHP: 8.1.6
 - PHPCS: 3.7.0
 - Standard: PSR12

**Comment by jakubkosinski99** [comment]
I noticed that too. Nothing can be done

**Comment by Arthur-Sk** [comment]
Same.
```php
// @codingStandardsIgnoreStart
public function empty() {
  // @codingStandardsIgnoreEnd
```
As a temporary workaround.

**Comment by dragosprotung** [comment]
This actually is a problem for more function names:

```php
<?php

declare(strict_types=1);

final class Test
{
    public function unset(): self
    {
        return new self();
    }

    public function empty(): self
    {
        return new self();
    }

    public function isset(): self
    {
        return new self();
    }
}
```

```
--------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 3 LINES
--------------------------------------------------------------------------
  7 | ERROR | Expected "function abc(...)"; found "function abc(...)"
    |       | (Squiz.Functions.FunctionDeclaration.Found)
 12 | ERROR | Expected "function abc(...)"; found "function abc(...)"
    |       | (Squiz.Functions.FunctionDeclaration.Found)
 17 | ERROR | Expected "function abc(...)"; found "function abc(...)"
    |       | (Squiz.Functions.FunctionDeclaration.Found)
--------------------------------------------------------------------------
```
